app/layout.tsx

```diff
@@ -27,6 +27,17 @@ export default function RootLayout({
 
   return (
     <html lang="en">
+      <head>
+        <link
```

Comment:
@andrewebdev - I'm curious about whether it would be more performant to use svgs instead of loading an external resource. I was reading about `@mui/icons-material` [here ](https://mui.com/material-ui/icons/)and wondering if that would be a good option.

Reply:
I have a couple of opinions on this (and I'm happy to have them challenged :wink: )

1. I chose the font version instead of the SVG version because it's a standard that many other platforms started adopting
2. We can download the font and load it locally, just like the other ones you pointed out.
3. I normally start out loading fonts direct from google and make use of their CDN. Performance is rarely an issue, but when it becomes a problem, that's when I download the font locally.
4. I would rather use the font library of the icons, over another third party that wraps them. It introduces yet another dependency (and whatever other dependencies they import), and makes security more fickle down the line.

I'm happy to be challenged on these points, and even though I feel strongly about it, I'll change it if you want me to :)

Reply:
@andrewebdev let me do more reading on this, but I always thought using a handful of svgs was better than loading the whole font family.

Reply:
@andrewebdev I've read some interesting articles on svgs vs icon fonts. In general, it seems like more developers are leaning towards svgs because:
1) loading icon fonts could result in some unexpected flashing
2) svgs are more accessible
3) When a visitor downloads your icon font, they are downloading every icon , even though you only want to use a handful of them
4) visual quality for svgs are better than icon fonts due to anti-aliasing in icon fonts
5) icon fonts are more difficult to style and position

One drawback of using svgs though is that inline svgs cannot be cached, but external ones can.

Here are links to some articles I read:
https://www.keycdn.com/blog/icon-fonts-vs-svgs
https://www.irigoyen.dev/blog/2021/02/17/stop-using-icon-fonts/
https://getdevdone.com/blog/icon-fonts-vs-svg-icons.html

Let me know your thoughts

Reply:
Okay some are valid points. I've not noticed any issues with using icon fonts in the past, but I do load these things differently in my projects. The anti-aliasing is the most compelling argument. I've not run into that myself, but I can see how that could be an issue on some systems.

I still don't think it's worth importing an entire third party library just for icons, so I'll rather download the google material icons (they are available as SVG's) and I'll use that in our `dmpicon` component.

Reply:
@andrewebdev that sounds great. Thanks Andre.